### DISCUSSION TITLE
Reasons for exclusion

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#697] [Q&A] [answered]
We have seen that in production there are shipments in which the description of the reasons for exclusion indicates the description of the selected reason in the description of the reasons for exclusion.

[https://ted.europa.eu/udl?uri=TED:NOTICE:593281-2023:TEXT:EN:HTML](url)
![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/83206179-491c-453d-aa7b-caabd445094f)

While others send a free text description. 

[https://ted.europa.eu/udl?uri=TED:NOTICE:589949-2023:TEXT:EN:HTML](url)
![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/7dff4b68-9bc4-460a-92ac-b2bfd31a8cd4)

Which option would be more correct?

Thanks in advance

**Comment by ferrakaop** [comment, marked as answer]
This will depend on the buyer and the guidance in their organisation or the legislation in their country. Please consult your business colleagues - this isn't a technical question about the SDK.
